src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -1012,7 +1010,11 @@ const Authenticate = (() => {
         if (is_required || has_svg_account){
             initTab();
             initAuthentication();
-            if (is_from_mt5) {
+
+            const { identity, document } = authentication_status;
+            const is_not_fully_authenticated = identity.status !== 'verified' && document.status !== 'verified';
+            const is_not_high_risk = !/high/.test(State.getResponse('get_account_status.risk_classification'));
+            if (is_not_fully_authenticated && has_svg_account && is_not_high_risk) {
```

Comment:
this validations are from previous commits.  whether to add the banner or not is still under discussion